src/Compilers/CSharp/Portable/Utilities/IValueSet.cs

```diff
@@ -51,6 +53,11 @@ internal interface IValueSet
         /// Does this value set contain no values?
         /// </summary>
         bool IsEmpty { get; }
+
+        /// <summary>
+        /// Produce a sample value contained in the set. Throws <see cref="ArgumentException"/> if the set is empty.
```

Comment:
>`Throws <see cref="ArgumentException"/> if the set is empty.`

there's some left over of ArgumentException in a few overrides too (still throwing).